backend/src/test/resources/Subject-Hub.postman_collection.json

```diff
@@ -182,6 +182,160 @@
+                    "response": []
+                },
+                {
+                    "name": "Get Assessments (w GET params)",
+                    "request": {
+                        "url": {
+                            "raw": "{{domain}}/api/universities/university/1/subjects/subject/1/assessments?",
+                            "host": [
+                                "{{domain}}"
+                            ],
+                            "path": [
+                                "api",
+                                "universities",
+                                "university",
+                                "1",
+                                "subjects",
+                                "subject",
+                                "1",
+                                "assessments"
+                            ],
+                            "query": [
+                                {
+                                    "key": "name",
+                                    "value": "Assessment task 1",
+                                    "equals": true,
+                                    "description": "",
+                                    "disabled": true
+                                },
+                                {
+                                    "key": "weighting",
+                                    "value": "30",
+                                    "equals": true,
+                                    "description": "",
+                                    "disabled": true
+                                },
+                                {
+                                    "key": "type",
+                                    "value": "PROJECT",
+                                    "equals": true,
+                                    "description": "",
+                                    "disabled": true
+                                },
+                                {
+                                    "key": "groupWork",
+                                    "value": "TRUE",
+                                    "equals": true,
+                                    "description": "",
+                                    "disabled": true
+                                }
+                            ],
+                            "variable": []
+                        },
+                        "method": "GET",
+                        "header": [],
+                        "body": {
+                            "mode": "raw",
+                            "raw": ""
+                        },
+                        "description": "Fetches all assessments for a subject with GET params that you can enable. Select \"Params\" next to the address and enable the ones that you wish to use."
+                    },
+                    "response": []
+                },
+                {
+                    "name": "Get Assessments (w GET params and Fuzzy examples)",
```

Comment:
You don't need one with fuzzy examples. You're not doing anything different with them.